pkg/quota/interfaces.go

```diff
@@ -50,13 +50,15 @@ type Evaluator interface {
 	// If true, admission control must perform quota processing for the operation, otherwise it is safe to ignore quota.
 	Handles(operation admission.Attributes) bool
 	// Matches returns true if the specified quota matches the input item
-	Matches(resourceQuota *api.ResourceQuota, item runtime.Object) (bool, error)
+	Matches(resourceQuota *api.ResourceQuota, item runtime.Object, matchPriorityClassScopes bool) (bool, error)
 	// MatchingResources takes the input specified list of resources and returns the set of resources evaluator matches.
 	MatchingResources(input []api.ResourceName) []api.ResourceName
 	// Usage returns the resource usage for the specified object
 	Usage(item runtime.Object) (api.ResourceList, error)
 	// UsageStats calculates latest observed usage stats for all objects
 	UsageStats(options UsageStatsOptions) (UsageStats, error)
+	// GetPriorityClass returns PriorityClassName, if assigned any, to the object
+	GetPriorityClass(item runtime.Object) (*string, error)
```

Comment:
i also dont understand why this is required?  this should be scoped to the pod evaluator only.